meteor/client/ui/Shelf/Inspector/ItemRenderers/ActionItemRenderer.tsx

```diff
@@ -258,13 +233,7 @@ export default translateWithTracker<IProps, {}, ITrackedProps>((props: IProps) =
 							{action.userDataManifest && action.userDataManifest.editableFields && !targetAction ? (
 								<Spinner />
 							) : action.userDataManifest && action.userDataManifest.editableFields && targetAction ? (
-								<>
-									{this.renderConfigFields(
-										action.userDataManifest.editableFields,
-										targetAction,
-										'transformedUserData.'
-									)}
-								</>
+								<span>Editable Fields are not currently supported.</span>
 							) : null}
 						</div>
```

Comment:
I guess the "Editable Fields ..." text should be passed through the `t` function, but more importantly: does this section make any sense now? Should we just not show anything, instead of showing a spinner only to then later show "Editable Fields are not currently supported"?

Reply:
I suggested doing this:

> But I think it would be better to leave the current ui bits in place, and to replace some of the inside of renderConfigFields with something like <span>Not implemented!</span>, so that when someone tries to use the editableFields property next they don't have to spend time trying to figure out why it isn't working and are instead reminded that it isnt implemented.